webpack.config.js

```diff
@@ -22,14 +22,20 @@ module.exports = {
                 },
             },
             {
-                test: /\.css$/,
+                test: /\.s[ac]ss$/,
                 use: [
                     {
-                        loader: 'css-loader'
+                        loader: 'style-loader'
                     },
                     {
-                        loader: 'style-loader'
+                        loader: 'css-loader'
                     },
+                    {
+                        loader: 'sass-loader',
+                        options: {
+                            implementation: require("sass"),
```

Comment:
Add a comment here to specify that this is Dart SASS
```suggestion
                           // specified that Dart SASS should be used
                            implementation: require("sass"),
```

Reply:
Dealt with in [7e81e7c](https://github.com/SaileshPatel/productivity-app-react-custom-build/pull/2/commits/7e81e7cd1d57a9f7a18ac11ac6a03e46427e9db2)